src/components/header/header.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Logo from './components/Logo';
+import { HeaderContainer, Nav } from './styles';
+import Menu from './components/Menu';
+
+const Header = () => {
+  return (
+    <HeaderContainer>
+      <Nav>
+        <Logo />
```

Comment:
The logo is not matching the design completely

![image](https://user-images.githubusercontent.com/9722792/77818689-c2b15280-70d4-11ea-95e3-41243d0cdc14.png)